src/gwt/src/org/rstudio/core/client/widget/ModifyKeyboardShortcutsWidget.java

```diff
@@ -1450,6 +1466,8 @@ public Object getKey(KeyboardShortcutEntry item)
    private List<KeyboardShortcutEntry> originalBindings_;
    private Pair<Integer, Integer> lastSelectedIndices_;
    private boolean swallowNextKeyUpEvent_;
+
+   private ThemedButton applyButton_;
```

Comment:
This change (having the apply button be disabled initially) causes a change in behavior that I suspect will result in negative feedback from some users. Previously you could type a new shortcut sequence and immediately click the Apply button; now you have to either hit the Enter key or click outside the text field to enable the Apply button. What you did is reasonable, but I'd be inclined to leave this behavior the way it was.

Reply:
This behavior has been reverted, but I have left in the reference to `applyButton_` in case we ever change our minds about this.